docs/development_suite/identity-and-access-management/service-account-management.md

```diff
@@ -0,0 +1,82 @@
+- Authentication method: it is necessary to verify the identity of the Service Account and can be of two types <!-- TODO: SCREENSHOT OF THE DIFFERENT FLOWS -->
+    - `client_secret_basic`: the Service Account authenticates by presenting its `client_id` and `client_secret` in the Authorization header of the request, in the format `Authorization: Basic <base64 encoded client_id:client_secret>`. The server then decodes the header and validates the credentials against its records to authenticate the client.  
+    - `private_key_jwt`: the Service Account authenticates by signing a JWT (JSON Web Token) using its private key. The client includes a JWT in the `Authorization` header of the request, with specific claims set to appropriate values. The server then verifies the JWT by validating the signature using the client's public key, and checking that the claims are valid and match its records.  
+
+For further information on how to perform OAuth2 compliant client credential flows, check out the [Client Credentials Service documentation](../../runtime_suite/client-credentials/configuration.md)
+
+### Editing a Service Account Role
+
+A Service Account Role in the Company can be modified: to do so, simply click on the pencil-shaped button on the desired Service Account row and select the new Role.
+
+<!-- TODO: SCREENSHOT OF COMPANY EDIT SA MODAL -->
+
+### Removing a Service Account from the Company
+
+A Service Account can be removed from the Company by clicking the trash icon on the table and confirming the action.
+
+<!-- TODO: SCREENSHOT OF COMPANY DELETE SA MODAL -->
+
+:::warning
+Removing a Service Account from the Company will permanently delete the account and all its existing Roles, which will be lost and cannot be recovered.
+:::
+
+## Managing Project and Environment Service Accounts
+
+A User with enough administrative permission on a specific Project will be able to view all the existing Service Accounts in the Company and edit their Role on the specific Project (and, optionally, on each existing Runtime Environment).
+
+<!-- TODO: ADD SCREENSHOT OF PROJECT ADMIN PORTAL FILTERED BY SA IDENTITY TYPE -->
+
+:::note
+Although the Project Identities administration portal shows all the Company's identities, this does not mean that all of the identities have access to the Project, since this depends on the Role they are assigned in the Company and how the permissions are inherited.
+
+For further information about permissions and Role inheritance check out the [Console Levels and Permission Management](/development_suite/identity-and-access-management/console-levels-and-permission-management.md) page.
+:::
+
+### Editing a User Role at Project level
+
+A User Role in the Project or any of the Project's Runtime Environments can be modified. To do so, just open the editing dialog and select the proper Role for the Project itself or for each Runtime Environment.
+
+<!-- TODO: SCREENSHOT OF COMPANY EDIT SA MODAL -->
```

Comment:
below here i'd place a `##` with information about how a service account should authenticate to the Console and how to perform an example request on some endpoint.

endpoint: /api/m2m/oauth/token

with a request example and a link for "more information" targeting the client credential usage docs